dist/api_data.py

```diff
@@ -304,8 +304,7 @@ def __ge__(self, other):
     Config('huffman_key', 'none', r'''
         This option is no longer supported, retained for backward compatibility'''),
     Config('huffman_value', 'none', r'''
-        configure Huffman encoding for values. Permitted values are \c "none", \c "english",
-        \c "utf8<file>" or \c "utf16<file>". See @ref huffman for more information'''),
+        This option is no longer supported, retained for backward compatibility'''),
```

Comment:
I think we can add `undoc=True` if we no longer want to expose the API to future users.

Reply:
Good idea - done in e88e20d.